packages/anasazi/src/AnasaziTraceMinRitzOp.hpp

```diff
@@ -89,7 +105,7 @@ template <class Scalar, class MV, class OP>
 class TraceMinProjOp
 {
   typedef Anasazi::MultiVecTraits<Scalar,MV>    MVT;
-  const Scalar ONE; 
```

Comment:
@mhoemmen  I'm probably just going to close this pull request and re-do it from the other branch off my fork, which is a bit cleaner on my end.  I only saw about 2 or 3 files that had had whitespace stripped off a few source lines.  I have it cleaned up on my space and am just running the sems checkin-test script.  That one should hopefully have just comment lines affected.